examples/tv-app/linux/include/application-basic/Application.h

```diff
@@ -0,0 +1,34 @@
+/*
+ *
+ *    Copyright (c) 2021 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#pragma once
+
+#include "gen/enums.h"
+#include <cstdint>
+#include <string>
+
+struct Application
+{
+    std::string vendorName               = "\0";
```

Comment:
I think std::string has a default constructor, no need to initialize it this way. Also "\0" seems to be a double-0 which is likely not what is intended.

Are we using std::string because this is a linux-only example (hence heap is ok)?

Reply:
It was due to inipp library which forced me to use std::string. Anyhow I've returned back to char [32], which is per spec.